source/reporting/index.html.md.erb

```diff
@@ -111,7 +117,14 @@ For example:
 }
 ```
 
-You should receive the payment in your bank account within 2 business days of `captured_date`.
+If your PSP is Stripe, you should receive the payment in your bank account within:
+
+- 2 working days of `captured_date` if your user completed their payment during the week
+- 3 working days of `captured_date` if your user completed their payment at the weekend or on a bank holiday
+
+You can confirm when payments are paid into your bank account by signing in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) and going to __My Services__ > __View payments to your bank account__.
```

Comment:
Do we want to say 'when payments are paid into your bank account' or 'when transactions are paid into your bank account'? We're talking here about tracking the status of a transaction, so maybe that makes more sense? 

Also need to edit this to include the API report as well when that is released.

Reply:
@miriamalice we speak about payments in this section, so I think it's ok? Also, to check, are you referring to the https://github.com/alphagov/pay-tech-docs/pull/460 work?

Reply:
I think payments means different things which can be a bit confusing. 
Payments into a bank account sounds like it means the total amount that is paid into a bank account - the 'payment' that you have received from Stripe.
Here I think we mean, you confirm when individual payments have been paid into your bank account.

So it might be slightly confusing but also I think it's fine. Had forgotten we use 'payments' rather than 'transactions'.

And yes, referring to 460.

Reply:
ok, i'll add to the existing 460 PR to do this.